respect-datalayer/src/commonMain/kotlin/world/respect/datalayer/school/writequeue/WriteQueueItem.kt

```diff
@@ -22,7 +22,7 @@ class WriteQueueItem(
         val flag: Int
     ) {
         PERSON(1), CLASS(2), ENROLLMENT(3), PERSON_PASSWORD(4),
-        ASSIGNMENT(5), SCHOOL_APP(6);
+        ASSIGNMENT(5), SCHOOL_APP(6), PERSON_QRCODE(7), PERSON_QRCODE_DELETE(8);
```

Comment:
Deleting data must be done using ```StatusEnum``` (active or to be deleted) as per all other data.

PERSON_QRCODE_DELETE must be removed from here.

Reply:
To delete data you need to store it with the status set to be deleted. This is essential to allow systems that work offline to synchronize / other devices to understand what has been deleted.